src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientGameOverSystem.java

```diff
@@ -56,16 +56,17 @@
     /**
      * System to show game over screen once a team achieves goal score.
      *
-     * @param event  the event
+     * @param event the event
```

Comment:
I realize you just caught this line by fixing an errant space, but now that you've touched it anyway want to take a quick stab at adding a few more words for each param to make them more meaningful? 😁

Reply:
Yes, will do so!